src/main/kotlin/lotto/domain/WinningLotto.kt

```diff
@@ -0,0 +1,9 @@
+package lotto.domain
+
+class WinningLotto(val winningNumbers: LottoTicket, val bonusNumber: LottoNumber) {
+    init {
+        require(!winningNumbers.contains(bonusNumber)) {
+            throw IllegalArgumentException("보너스 번호는 당첨 번호와 중복될 수 없다.")
+        }
```

Comment:
`require()`는 값을 만족하지 않으면 `IllegalArgumentException`을 발생시킵니다.

```suggestion
        require(!winningNumbers.contains(bonusNumber)) {
            "보너스 번호는 당첨 번호와 중복될 수 없다."
        }
```